src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -103,12 +107,33 @@ const CooperationDetails = () => {
     dispatch(setIsActivityCreated(true))
   }, [dispatch])
 
+  const openAddReviewModal = useCallback(() => {
+    if (!cooperation || isLoading) return
```

Comment:
use braces please, and add space between `if` statement and the rest of code
```suggestion
    if (!cooperation || isLoading) {
       return
    }
    
```